__tests__/app.test.js

```diff
@@ -34,15 +34,35 @@ describe("testing app GET /api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
-        const topic = body;
-        expect(topic).toHaveLength(3);
-        expect(topic).toEqual(
+        const topics = body;
+        expect(topics).toHaveLength(3);
+        expect(topics).toEqual(
           expect.arrayContaining([expect.objectContaining(testTopicObj)])
         );
       });
   });
 });
 
+describe('testing app GET /api/users', () => {
+    const testUserObj = {
+        username: expect.any(String),
+        name: expect.any(String),
+        avatar_url : expect.any(String)
+    }
+    test('should return an array of user object with correct properties and satus 200', () => {
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({body}) => {
+            const users = body;
+            expect(body).toHaveLength(4);
+            expect(users).toEqual(
+                expect.arrayContaining([expect.objectContaining(testUserObj)])
```

Comment:
Be careful when using arrayContaining. It will also match the received array, if there are values in there that we don't expect. A forEach( ) would work here and avoid that issues

Reply:
Good to know; I'be thought about using forEach instead,